charts/gatekeeper/templates/gatekeeper-audit-deployment.yaml

```diff
@@ -33,7 +33,8 @@ spec:
         {{- toYaml .Values.auditPodAnnotations | trim | nindent 8 }}
         {{- end }}
       labels:
-{{- include "gatekeeper.podLabels" . }}
+        {{- include "gatekeeper.podLabels" . | nindent 8 }}
```

Comment:
Thanks for the PR!
Please remove any updates to the charts/ directory since those get autogenerated when we cut a new release. Thanks!

Reply:
Thanks @ritazh  I have done this now